src/components/Radio.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <fieldset class="f-radio">
+    <input
+      v-model="newResourceType.abstract"
+      type="radio"
+      id="abstract-true"
+      name="abstract"
+      value="true"
+    >
+    <label for="abstract-true">Yes</label>
+    <input
+      v-model="newResourceType.abstract"
+      type="radio"
+      id="abstract-false"
+      name="abstract"
+      value="false"
+    >
+    <label for="abstract-false">No</label>
+  </fieldset>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import Utils from '@/utils/Utils';
+
+@Component
+export default class Toggle extends Vue {
```

Comment:
To me, it does not look like a generalized radio element.
(`newResourceType`, `abstract`)

Reply:
Also, it only allows for two hard-coded radio buttons.